extensions/wikia/Search/classes/Config.php

```diff
@@ -34,303 +38,345 @@ class Config
+		self::FILTER_CAT_VIDEOGAMES,
+		self::FILTER_CAT_ENTERTAINMENT,
+		self::FILTER_CAT_LIFESTYLE,
 	];
 
 	/**
 	 * Associates short key names with filter queries.
 	 * This approach doesn't support on-the-fly language fields.
 	 * We could still append a key in __construct() if it becomes an issue.
+	 *
 	 * @var array
 	 */
 	protected $filterCodes = [
-			self::FILTER_VIDEO => '(is_video:true AND -is_image:true)',
-			self::FILTER_IMAGE => '(is_image:true AND -is_video:true)',
-			self::FILTER_HD    => 'video_hd_b:true',
+		self::FILTER_VIDEO => '(is_video:true AND -is_image:true)',
+		self::FILTER_IMAGE => '(is_image:true AND -is_video:true)',
+		self::FILTER_HD => 'video_hd_b:true',
 	];
 
 	/**
 	 * This is used to keep non-keyed filter queries unique in Solarium
+	 *
 	 * @var int
 	 */
 	public static $filterQueryIncrement = 0;
 
 	/**
 	 * Filter queries stored by "key"
 	 * Separate from traditional storage because the requirements are a bit more complex
+	 *
 	 * @var array
 	 */
 	protected $filterQueries = [];
 
 	/**
 	 * If a query matches an article, it may be stored here.
+	 *
 	 * @var \Wikia\Search\Match\Article
 	 */
 	protected $articleMatch;
 
 	/**
 	 * If a query matches a wiki, it may be stored here.
+	 *
 	 * @var \Wikia\Search\Match\Wiki
 	 */
 	protected $wikiMatch;
 
 	/**
 	 * If an error occurred during search, we store it here.
+	 *
 	 * @var \Exception
 	 */
 	protected $error;
 
 	/**
 	 * When set to true, we don't use boost functions in our query
+	 *
 	 * @var bool
 	 */
 	protected $skipBoostFunctions = false;
 
 	/**
 	 * Allows us to tell the factory which service we want
+	 *
 	 * @var string
 	 */
 	protected $queryService;
 
 	/**
 	 * Used to shift all MediaWiki logic elsewhere.
+	 *
 	 * @var MediaWikiService
 	 */
 	protected $service;
 
 	/**
 	 * Allows us to specify the use case for Wikia\Search\QueryService\Select\Dismax\CombinedMedia
 	 * By default, it's video-only. We can include images by changing this value to false.
+	 *
 	 * @var bool
 	 */
 	protected $combinedMediaSearchIsVideoOnly = true;
 
 	/**
 	 * Allows us to specify the use case for Wikia\Search\QueryService\Select\Dismax\CombinedMedia
+	 *
 	 * @var bool
 	 */
 	protected $combinedMediaSearchIsImageOnly = false;
 
 	/**
 	 * Specify minimal number of articles needed for wiki to appear in search results
+	 *
 	 * @var int
 	 */
 	protected $xwikiArticleThreshold = 50;
 
 	/**
 	 * Constructor method
+	 *
 	 * @param array $params
 	 */
 	public function __construct( array $params = [] ) {
 
 		$dynamicFilterCodes = [
-				self::FILTER_CAT_VIDEOGAMES    => Utilities::valueForField( 'categories', 'Games', [ 'quote'=>'"' ] ),
-				self::FILTER_CAT_ENTERTAINMENT => Utilities::valueForField( 'categories', 'Entertainment' ),
-				self::FILTER_CAT_LIFESTYLE     => Utilities::valueForField( 'categories', 'Lifestyle'),
-				];
+			self::FILTER_CAT_VIDEOGAMES => Utilities::valueForField( 'categories', 'Games', [ 'quote' => '"' ] ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "categories" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)